docs/content/scripting.md

```diff
@@ -40,7 +40,7 @@ That's all there is to a simple Rhai script.
 From a template, we can then use this script like this:
 
 ```
-{{ echo "world" }}
+\{{ echo "world" }}
```

Comment:
This should stay as `{{ echo "world" }}` so that the rhai script actually executes, right?

Reply:
The script is not expected to execute there as it demonstrates how the script can be called within a template.

Reply:
Yes as the shortcodes are not enabled for the scripting content. Also, the Rhai script isn't supposed to execute as it is just there to demonstrate how a script can be called form the template.

Reply:
I think since this markdown doesn't have `enable_shortcodes = true`, we should leave this example unescaped.  Because currently the rendered docs show this:

<img width="495" alt="Screen Shot 2022-07-29 at 11 03 35 AM" src="https://user-images.githubusercontent.com/9322017/181809065-46ed48f2-7642-4068-bde5-4b621e60d22a.png">

But don't we want it to show the following?

<img width="524" alt="Screen Shot 2022-07-29 at 11 04 21 AM" src="https://user-images.githubusercontent.com/9322017/181809179-92ad814c-33da-4c97-ac9e-7952df45dad9.png">

Reply:
Yes that is correct, I will make the change.